src/javascript/app_2/App/Components/Elements/Errors/error_component.jsx

```diff
@@ -2,17 +2,32 @@ import PropTypes    from 'prop-types';
 import React        from 'react';
 import IconError    from 'Assets/Common/icon_error.jsx';
 import { localize } from '_common/localize';
+import Localize     from '../localize.jsx';
 
-const ErrorComponent = ({ type, message }) => (
-    <div className='error-container'>
-        <IconError type={type} />
-        <p>{message || localize('Sorry, an error occured while processing your request.')}</p>
-    </div>
-);
+const ErrorComponent = ({ type, message }) => {
+    let msg = '';
+    if (typeof message === 'object') {
+        msg = <Localize
+            str={message.str}
+            replacers={message.replacers}
+        />;
+    } else {
+        msg = message;
```

Comment:
Question, `msg` in the else block is not `localized`. Are we counting on the user to send the `localized` string?

Reply:
those messages are `localized` before it is being passed to this component